platform/podjs/src/manifest.ts

```diff
@@ -0,0 +1,63 @@
+import * as Decoder from "io-ts/Decoder";
```

Comment:
@JJ refactored whole file addressing the issues you mentioned above.
Let me know if LGTY or we can review it together if you fancy :)

Reply:
Checking it out now